docs/css/fiori-fundamentals.css

```diff
@@ -3549,7 +3549,7 @@ This will render 4 boxes spanning 2 cols each indented 2 cols
       -webkit-box-shadow: 0 0 0 1px var(--fd-color-action-focus);
               box-shadow: 0 0 0 1px var(--fd-color-action-focus); }
     [dir="rtl"] .fd-alert__close, .fd-alert__close[dir="rtl"] {
-      left: 4px;
+      left: 0;
```

Comment:
I think this PR should be held until #1345 is merged.  The files `fui-site.css` and `fiori-fundamentals.css` are no longer required to be updated in source control (since they are generated).

Reply:
#1345 is merged :)